.github/workflows/ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Nginx image creator
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+
+  create-and-push-image:
+    runs-on: ubuntu-latest
+
+    permissions:
+      contents: read
+      packages: write
+      attestations: write
+      id-token: write
+
+    steps:
+    - name: CHECKOUT
```

Comment:
Let's define task steps in lower case, i.e. `Checkout` or `Install node` and etc. Ususally upper case is used for env variables or secrets